enarx-mirrorlabels

```diff
@@ -0,0 +1,173 @@
+        'color': 'f98918',
+        'old_names': {},
+    },    
+    'debt': {
+        'description': 'Issues to deal with later',
+        'color': '0c0c75',
+        'old_names': {},
+    },    
+    'documentation': {
+        'description': 'Improvements or additions to documentation',
+        'color': 'efef15',
+        'old_names': {}
+    },    
+    'duplicate': {
+        'description': 'This issue or pull request already exists',
+        'color': 'cfd3d7',
+        'old_names': {},
+    },    
+    'enhancement': {
+        'description': 'New feature or request',
+        'color': 'a2eeef',
+        'old_names': {},
+    },    
+    'expertise needed': {
+        'description': 'This needs special attention from an area specialist.',
+        'color': 'f9928e',
+        'old_names': {},
+    },    
+    'good first issue': {
+        'description': 'Good for newcomers',
+        'color': 'a8f475',
+        'old_names': {},
+    },    
+    'help wanted': {
+        'description': 'Extra attention is needed',
+        'color': '008672',
+        'old_names': {},
+    },    
+    'host-components': {
+        'description': 'Components that live on the host, but not in Keeps',
+        'color': 'f22933',
+        'old_names': {},
+    },    
+    'ibm pef': {
+        'description': 'Issues related to IBM PEF',
+        'color': '0043ce',
+        'old_names': {},
+    },    
+    'infrastructure': {
+        'description': 'Improvements or additions to project infrastructure',
+        'color': 'ffdda8',
+        'old_names': {},
+    },    
+    'intel sgx': {
+        'description': 'Issues related to Intel SGX',
+        'color': '0071c5',
+        'old_names': {},
+    },    
+    'invalid': {
+        'description': "This doesn't seem right",
+        'color': 'e4e669',
+        'old_names': {},
+    },    
+    'mentorship': {
+        'description': 'A request for mentorship on the project.',
+        'color': '0e8a16',
+        'old_names': {},
+    },    
+    'meta': {
+        'description': 'Larger project tasks and goals',
+        'color': '149e7b',
+        'old_names': {},
+    },    
+    'question': {
+        'description': 'Further information is requested',
+        'color': 'd876e3',
+        'old_names': {},
+    },    
+    'research': {
+        'description': '',
+        'color': 'ed8661',
+        'old_names': {},
+    },    
+    'security': {
+        'description': 'Issues that have security implications',
+        'color': 'ff0000',
+        'old_names': {},
+    },    
+    'syscall': {
+        'description': 'syscall-related issues and PRs',
+        'color': 'a51f3c',
+        'old_names': {},
+    },    
+    'wasm': {
+        'description': 'Issues related to WebAssembly',
+        'color': '654EF0',
+        'old_names': {},
+    },    
+    'wontfix': {
+        'description': 'This will not be worked on',
+        'color': 'ffffff',
+        'old_names': {},
+    }
+}
```

Comment:
I have now reviewed the debug output for this script and I think we need to make this less monolithic. I think we need different kinds of labels here. Some are organization wide. Some are repo-specific.